### PR TITLE
Fixed issue regarding Past Events Blog pages

### PR DESCRIPTION
**Pull request by suneel87**
Signed-off-by: Suneel Nadipalli <nsuneel89@gmail.com>
Fixes 1
- Removed Next Event Button from cultural week blog page
Fixes 2
- Centered day 1, day 2, etc, in schedule page

### CODE REVIEW

File: functions/views/culturalWeek.ejs [hunk 1]
```diff
@@ -205,8 +205,8 @@
 								<h5>Management Week</h5>
 							</a>
 						</div>
-						<div
-							class="blog-single-content__paging__divide blog-single-content__paging__divide--next"
+<!--						<div
+							class="blog-single-content__paging__divide blog-single-content__paging__divide&#45;&#45;next"
```
Comment (aravindvnair99) [inline functions/views/culturalWeek.ejs:209]:
```suggestion
							class="blog-single-content__paging__divide blog-single-content__paging__divide--next"
```

@suneel87 Don't use HTML entities here. If you have some extension for that, disable it as it's clearly not working correctly.

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]

File: functions/views/culturalWeek.ejs (hunk 1 above)
Comment (suneel87) [reply functions/views/culturalWeek.ejs:209]: Wha...I don't remember putting that there..I didn't even touch that, all I did was comment out a portion of the code

File: functions/views/schedule.ejs [hunk 2]
```diff
@@ -219,7 +219,7 @@
 					<!-- ---------------------------------------------------------------------------------- -->
```
Comment (aravindvnair99) [inline functions/views/schedule.ejs:219]:
```suggestion
```

File: functions/views/schedule.ejs [hunk 3]
```diff
@@ -397,7 +397,7 @@
 					</div>
 					<!-- ---------------------------------------------------------------------------------- -->
```
Comment (aravindvnair99) [inline functions/views/schedule.ejs:398]:
```suggestion
```

**Review by aravindvnair99 — DISMISSED** [review]

**Review by aravindvnair99 — APPROVED** [review]

**Pull request merged**